inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/settings/WorkflowAndMonitoringPanel.properties

```diff
@@ -14,15 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-workflow=<strong>Please select a workflow manager</strong>
+panel=panel
+workflow=Please select a workflow manager
 defaultWorkflow=Default workflow manager
 
-monitoring=<strong>Please select a monitoring page</strong>
+monitoring=Please select a monitoring page
 defaultMonitoring=Default monitoring page
 
 #Card Header
-Monitoring=<strong>Monitoring</strong>
-Workflow=<strong>Workflow</strong>
-Settings=<strong>Settings</strong>
+Monitoring=Monitoring
+Workflow=Workflow
+Settings=Settings
 
 confirm=confirm
```

Comment:
We typically call it "save" and it is a globally available string so you don't have to define it here in the properties.